### DISCUSSION TITLE
Domain name renewal

### DISCUSSION THREAD

**Discussion by celyes** [softvenue/i18n#81] [General]
<p align="right" direction="rtl">
لقد لاحظت اقتراب انتهاء صلاحية اسم النطاق و اردت التنبيه بهذا الأمر.
</p>
<p align="right">
@Aissaoui-Ahmed 
ألقِ نظرة من فضلك
</p>

**Comment by Aissaoui-Ahmed** [comment]
<div dir=rtl > شكرا @celyes على التذكير لقد تم التجديد </div>

![image](https://user-images.githubusercontent.com/20658625/111902171-509bf100-8a3c-11eb-8909-1b0e30e115a3.png)
